metagraph/integration_tests/test_api.py

```diff
@@ -183,15 +187,17 @@ def test_api_raw_search_empty_fasta_desc(self):
         self.assertEqual(ret[0]['seq_description'], '')
 
 
+@parameterized_class(('mode',), input_values=[('canonical',), ('primary',)])
 class TestAPIClient(TestAPIBase):
     graph_name = 'test_graph'
 
     sample_query = 'CCTCTGTGGAATCCAATCTGTCTTCCATCCTGCGTGGCCGAGGG'
-    sample_query_expected_cols = 98
+    sample_query_expected_cols = 99
```

Comment:
is this supposed to be the expected number of columns which have a match to the query? It would be useful to either name this `sample_query_expected_match_cols` or write a comment describing this. The value is so close to the total number of columns that this might be suspected as a bug.

Reply:
Makes sense, I was a bit confused at first as well. Feel free to change.

Reply:
btw, it changed from 98 to 99 because of the switch from the basic to canonical/primary graphs.